kunstmaan/bundles-cms/4.0/etc/packages/kunstmaan_liip_imagine.yaml

```diff
@@ -0,0 +1,75 @@
+liip_imagine:
```

Comment:
Hi @fabpot 

This is a configuration that comes from our `StandardEdition`.  It's not critical to be there and your completely right that it's not the responsibility of this recipe, but it's a nice to start with this already configured so you don't have to anymore. My question is, how would you go about `suggested configuration` for a dependency like LiipImagine? Because when I want to use the CMS, I don't want to go to each individual dependency to search how I should configure it. Do you have some kind of solution/suggestion for this?